Feature/hr 100634472

zehash opened the pull request:
Edit host profile page to read easier.

https://www.pivotaltracker.com/n/projects/1108894/stories/100634472

Started refactoring the front-end view for host profiles - Created a new Haml version of existing slim view. Needs plenty of work in terms of ui refactoring and arrangement. This is just an initial skeleton as requirements come through.
# Fixes feature <100634472>

zehash committed:
Edit host profile page to read easier.

https://www.pivotaltracker.com/n/projects/1108894/stories/100634472

Started the refactor of this page with new features such as a new gallery section and removal of redundant code.

#Fixes feature <100634472>

zehash committed:
Edit host profile page to read easier.

https://www.pivotaltracker.com/n/projects/1108894/stories/100634472

Started refactoring the front-end view for host profiles - Created a new Haml version of existing slim view. Needs plenty of work in terms of ui refactoring and arrangement. This is just an initial skeleton as requirements come through.

#Fixes feature <100634472>

zehash committed:
Edit host profile page to read easier.

https://www.pivotaltracker.com/n/projects/1108894/stories/100634472

Short term tweaks requested by Tom. Pitch to Investors/partners/interested parties in 4 days. Changes are small and needed fast. Sticking to slim for the primary view for now. A haml version was created and pushed earlier. That still needs work.

#Works on feature <100634472>

zehash committed:
Merge branch 'develop' of https://github.com/pethomestay/phs into feature/HR-100634472

zehash committed:
Work in progress - changed the Ui for the page -> removed redundant code ->  added sticky sidebar -> view shifted from tab based to scroll based control for the user.

Work still required on the scroll menu (sticky sidebar) when you scroll to the bottom of the page
Need to add more refined touch-ups.

https://www.pivotaltracker.com/n/projects/1108894/stories/100634472

#Work pending on feature <100634472>

zero51 commented on app/assets/javascripts/homestays/show.js:
```diff
@@ -0,0 +1,98 @@
+  $('.enquiry_check_out_date .input-group-addon').on('click', function(evt) {
+    evt.preventDefault();
+    check_in_picker.hide();
+    return check_out_picker.show();
```
I have to agree with Hound here - camel case for JS. Thoughtbot, as always, has a great summary (https://github.com/thoughtbot/guides/tree/master/style/javascript) and airbnb have a pretty goof detailed style guide (https://github.com/airbnb/javascript).

zero51 commented:
Looking good. Could you please add screenshots on UI pull requests?

zehash committed:
Work completed - changed the Ui for the page -> removed redundant code -> Sticky sideBar and new sticky top bar added

New views for lg, md, sm, xs - for the host profile page
New modal for mobile view pricing and services.
Bug fixes
Code refactoring
Huge overhaul on css files
add animation

https://www.pivotaltracker.com/n/projects/1108894/stories/100634472

#Work pending on feature <100634472>
